CONTRIBUTE.md

```diff
@@ -7,49 +7,87 @@
 ### Contribution of OEO content
 Please read [OEO wiki](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.
 
-### Philosophy
+### Workflow
 
 The development of a feature for this repository is inspired from the workflow described 
 by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
 
-1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository to describe the addition to the ontology. Discussion about the implementation details should occur within this issue 
+1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository
+
+    Choose the right issue for you among the choices you have on github:
+
+    A) Including a new term to the ontology. Only 1 new term (with a few subclasses, if at all)
+    
+    B) Restructuring existing parts of the ontology
+    
+    C) Other issue
+
+    Discussion about the implementation details should occur within this issue. [Assign a project](https://help.github.com/en/github/managing-your-work-on-github/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar) (default "Issues") to the issue
+
 2. Create a branch from dev to work on your issue (see below, the "Branch name convention" section)
     ```bash
     git checkout -b feature/myfeature dev
     ```
-    As all need to edit the same `oeo.omn` file, it is better to merge one additions as fast as possible to avoid merging conflicts. 
-    Therefore, it is best to create a pull request and merge it at the end of each work session if the ontology changed much.
-    This is different from a more conventional workflow where each issue is linked to only one pull request
+    As all need to edit the same `oeo.omn` file, it is better to merge one changes as fast *as possible* (i.e. do not wait for 6 months) to avoid merging conflicts
 3. Open [protégé](https://protege.stanford.edu/) and work on the ontology
-    Please refrain from editing the `oeo.omn` file in a text editor. 
-    If you do so, open protégé and check that the nodes which you modified do not have any problems
-4. Describe briefly what you changed in the `CHANGELOG.md` file and commit the file last
+
+    One can also edit the `oeo.omn` file in a text editor
+4. Before committing your changes, open the `oeo.omn` file with protégé and save the file from protégé. See the "Conventions" section for the commit messages format
```

Comment:
So no checking of problematic nodes necessary? If it was necessary, could you please describe how this should be done?

Reply:
It is described in the file `tests/InconsistencyTestGuide.md`, I should add a link to it. Or move the content of this file to the wiki and link to the wiki.

Reply:
Done in 867ec586d8b1c4550976321f73b870b77ef01667